### DISCUSSION TITLE
Opening <a> tag links in a new tab/window

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#2628] [Q&A] [answered]
I'm facing a challenge with opening links contained within <a> tags in a new tab or window. The <a> tags in the HTML markup do not have the target="_blank" attribute, and I'm looking for a reliable method to achieve this functionality. Thank you very much

**Comment by mdmintz** [comment, marked as answer]
This will make every link (`<a>` tag) on that page open in a new tab:

```python
sb.set_attributes('a', "target", "_blank")
```
